cluster/charts/universal-crossplane/templates/crossplane/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
         app: {{ template "name" . }}
         release: {{ .Release.Name }}
     spec:
+      securityContext:
```

Comment:
@muvaf do we have to add new values manually to are template values.yaml? (e.g. we have securityContext but not this new pod securityContext https://github.com/upbound/universal-crossplane/blob/6886043a3b74859308df7d2830ccc713d1e233e8/cluster/charts/universal-crossplane/values.yaml.tmpl#L49)

Reply:
Yes, pushed another commit to do it. Thanks for catching!